src/tracing/errors.ts

```diff
@@ -92,8 +90,18 @@ function shouldRejectError(error: EventHint['originalException']) {
       return true
     }
 
+    if ('stack' in error && typeof error.stack === 'string') {
+      // Errors coming from a browser extension can be ignored. These errors are usually caused by extensions injecting
+      // scripts into the page, which we cannot control.
+      if (error.stack.match(/-extension:\/\//i)) return true
+
+      // Errors coming from OneKey (a desktop wallet) can be ignored for now.
+      // These errors are either application-specific, or they will be thrown separately outside of OneKey.
+      if (error.stack.match(/OneKey/i)) return true
```

Comment:
super nit but if meant to be insensitive case may as well

```suggestion
      if (error.stack.match(/onekey/i)) return true
```